cilium-health/cmd/get.go

```diff
@@ -31,12 +33,14 @@ var healthGetCmd = &cobra.Command{
 				os.Exit(1)
 			}
 		} else {
+			//NodeEncription status
+			statusNodeEncryption := node.GetOptOutNodeEncryption()
```

Comment:
This will unfortunately not work.

`SetOptOutNodeEncryption` is called by the agent during boot up, if it detects that the local node needs to opt-out of encryption. This means that other processes, such as the cilium client cli, or the cilium-health cli, will not observe the value used by the agent if they read this value here.

What you will have to do instead is read out this value in the agent, and the expose it in the Cilium API as described in https://github.com/cilium/cilium/issues/24239

Reply:
Thank you, Sebastian, for the code review. I've fixed it.